monero-outreach-docs/translations/pt-br/quickfacts/QuickFacts_pt-br.md

```diff
@@ -0,0 +1,125 @@
+# MONERO
+
+Monero é uma moeda digital segura, privada, não-rastreável que está disponível para todos. As suas caracterísitcas o tornam uma moeda fungível.
+
+# FATOS RÁPIDOS
+
+Usuários podem gastar com segurança, ninguém consegue ver seu saldo ou rastrear suas atividades sem sua permissão.
+
+Revisado em 1/2/2019 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)
+
+- [Versão colorida](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_pt.pdf)
+
+[getmonero.org](https://getmonero.org/)

```

Comment:
I was checking and the website already had Brazilian portuguese available so I will suggest the link changes to the document.

New link:
https://www.getmonero.org/pt-br/